### DISCUSSION TITLE
Adding new languages

### DISCUSSION THREAD

**Discussion by fbollon** [MohmmedAshraf/laravel-translations#31] [Q&A] [answered]
Hi,

For the project I am working on the supported languages are the following:
        'en_EN',
        'de_DE',
        'fr_FR',
        'ko_KR',
        'ro_RO',
        'vi_VN',
        'zh_CN',

I executed the commands
`php artisan translations:install`
`php artisan migrate`
then I added in the table ltu_languages the above languages

then I ran with success
`php artisan translations:import`

but when I try to access the translations UI I get the error message :

Unable to locate a class or view for component [flag-language-en_EN].

is there any way to add languages not provided in the table ltu_languages ?

Thanks for your help

**Comment by MohmmedAshraf** [comment, marked as answer]
Hi @fbollon I will start working on fixing this next week stay tuned for updates
